bliss/encoder/image_normalizer.py

```diff
@@ -39,12 +43,31 @@ def __init__(
         self.clahe_min_stdev = clahe_min_stdev
         self.asinh_params = asinh_params
 
+        if self.asinh_params:
+            thresholds_num = len(self.asinh_params["thresholds"])
+            thresholds = (
+                torch.tensor(self.asinh_params["thresholds"])
+                .view(1, -1)
+                .expand(len(self.bands), thresholds_num)
+                .view(1, len(self.bands), thresholds_num, 1, 1)
+            ).clone()
```

Comment:
Statements that span multiple lines using parentheses are hard to read. (Better not to rely on the `black` formatter to reformat long lines.) Can you define some intermediate variables instead?

Reply:
use intermediate variables to simplify it